lib/MiqVm/MiqRhevmVm.rb

```diff
@@ -177,4 +182,19 @@ def unmount_storage
       $log.warn "#{log_header} Failed to unmount all items from <#{nfs_mount_root}>.  Reason: <#{$!}>"
     end
   end
+
+  private
+
+  # Output attributes of LV in column format and parse to retrieve active status
+  def lv_active?(lv_path)
+    AwesomeSpawn.run!("sudo lvs", :params => [:noheadings, :o, "lv_active", lv_path]).output.strip == "active"
+  end
+
+  def storage_type_block?(storage_type)
+    ["ISCSI", "fibrechannel"].include?(storage_type)
```

Comment:
I think it is "FCP" not "fibrechannel", https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/technical_reference/storage_domain_types1 

It _might_ be lowercase, in the Ovirt/RHV parsers we upcase it `storagedomain.dig(:storage, :type).upcase` and I see a bunch of places that .downcase before comparing.  I think the safest thing to do would be
```suggestion
    ["ISCSI", "FCP"].include?(storage_type.upcase)
```

And the same below with nfs / glusterfs

Reply:
Thanks